client/components/app.jsx

```diff
@@ -1,25 +1,18 @@
 import React from 'react';
-
+import Header from './header';
+import ProductList from './product-list';
 export default class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {
-      message: null,
-      isLoading: true
-    };
-  }
 
-  componentDidMount() {
-    fetch('/api/health-check')
-      .then(res => res.json())
-      .then(data => this.setState({ message: data.message || data.error }))
-      .catch(err => this.setState({ message: err.message }))
-      .finally(() => this.setState({ isLoading: false }));
   }
 
   render() {
-    return this.state.isLoading
-      ? <h1>Testing connections...</h1>
-      : <h1>{ this.state.message }</h1>;
+    return (
+      <div className="container">
+        <Header/>
+        <ProductList/>
+      </div>
```

Comment:
Your header shouldn't be in a container. That's why your page header/navbar thing isn't taking up the full width of the page like it should.